owdex/__init__.py

```diff
@@ -11,17 +12,28 @@
 from .usermanager import UserManager
 
 
-def create_app(config_dict=None):
+def create_app(config_dict={}, indices_dict={}):
     app = f.Flask("owdex")
 
     for file in ("../../owdex.toml", "/owdex.toml"):
         try:
             app.config.from_file(file, load=toml.load)
         except FileNotFoundError:
             pass
+    app.config = app.config | config_dict
 
-    if config_dict:
-        app.config = app.config | config_dict
+    indices = {}
+    for filename in ("../../indices.json", "/indices.json"):
+        try:
+            with open(filename) as file:
+                indices = json.load(file)
+        except FileNotFoundError:
+            print(f"not find {filename}")
+    indices = indices | indices_dict
+    if not indices:
+        raise RuntimeError("No indices file specified!")
```

Comment:
I largely just felt like using JSON because it represents nested lists more naturally. But you're right, it should be part of `owdex.toml`. It'll just look slightly less sexy 🤷🏻

Reply:
Addressed by [70755fe](https://github.com/owdex/owdex/pull/72/commits/70755fe21a763724e25a3435f836a9f0798008cd)